tests/src-android/cgeo/geocaching/storage/DataStoreTest.java

```diff
@@ -97,6 +97,66 @@ public static void testStoredLists() {
         }
     }
 
+    public static void testMoveToList() {
+
+        int listId1 = StoredList.STANDARD_LIST_ID;
```

Comment:
about the issue itself: took me a while to get the actual problem. Fix looks a bit strange to me, but it should be safe now